sagemaker_model_monitor/model_monitor_batch_transform/SageMaker-ModelMonitoring-Batch-Transform-Data-Quality-With-SageMaker-Pipelines-On-schedule.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Please only use present tense. Also, remember to change the word into plural form if the are more than one of it -&gt; "There will be two directory" -&gt; "There are two directories"</p><p>"the manifests will be containing" -&gt; "the manifests contain"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3639/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Doesn't seem to be addressed?</p>

Reply:
<p>Must address before merge!</p>

Reply:
done